ecs/cloudformation_test.go

```diff
@@ -573,7 +593,12 @@ func convertYaml(t *testing.T, yaml string, fn ...func(m *MockAPIMockRecorder))
 		aws: m,
 	}
 	template, err := backend.convert(context.TODO(), project)
-	assert.NilError(t, err)
+	if assertErr == nil {
+		assert.NilError(t, err)
+	} else {
+		assert.Equal(t, err.Error(), assertErr.Error())
```

Comment:
This will crash if `err` is nil, but I guess given it’s just in the tests that’s probably OK-ish?
I suppose an alternative would be to have `convertYaml` return `(*cloudformation.Template, error)` and have each test check the error accordingly.

Reply:
Actually this should probably be
```suggestion
		assert.Error(t, err, assertErr.Error())
```
but then maybe `assertErr` doesn’t need to be an error, it could just be a string?

Reply:
done!